### PR TITLE
Fixed EVO Coloring Bug for State

### PR DESCRIPTION
**Pull request by Domily11**

### CODE REVIEW

File: leaf-ui/js/displayAnalysis.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ function createSlider(currentAnalysis, isSwitch) {
     sliderObject.sliderElement.noUiSlider.on('update', function( values, handle ) {
         updateSliderValues(parseInt(values[handle]), currentAnalysis);
     });
-    EVO.setCurTimePoint(sliderMax, currentAnalysis);
+    EVO.setCurTimePoint(isSwitch ? 0 : sliderMax, currentAnalysis);
```
Comment (Manarnz) [inline leaf-ui/js/displayAnalysis.js]: remove extra space after `;`

File: leaf-ui/js/object/evoObjects.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ class EVO {
         else {
             console.log("ERROR: invalid sliderOption");
         }
-        EVO.refresh(analysisResult);
+        //EVO.refresh(analysisResult);
```
Comment (Manarnz) [inline leaf-ui/js/object/evoObjects.js]: delete it

**Review by Manarnz — APPROVED** [review]
It looks good! Great job Cat!!

**Comment by amgrubb** [comment]
Change Request:
- [X] This issue includes the problem that `revertNodeValuesToInitial` was never reimplemented.

```
// TODO: Re-write with new models
function revertNodeValuesToInitial() {
```
line ~742 in onFunctions.js

See also same comment in #525

When you navigate back to modeling mode, it should return to the initial values that are stored within the IntentionBBM -> userEvaluationList with absTime of 0.

**Review by amgrubb — CHANGES_REQUESTED** [review]
See comment above for requested additional feature.

File: leaf-ui/js/object/evoObjects.js [hunk 3]
```diff
@@ -141,42 +103,41 @@ class EVO {
...
+            }
+            EVO.changeIntentionsText(analysisResult);
+        } else {
+            EVO.returnAllColors(graph.getElements(), paper);
+            EVO.revertIntentionsText(graph.getElements(), paper);
                 EVO.revertIntentionsText(graph.getElements(), paper);    
-                    break;
+            EVO.revertIntentionsText(graph.getElements(), paper);
+                EVO.revertIntentionsText(graph.getElements(), paper);    
+            EVO.revertIntentionsText(graph.getElements(), paper);
```
Comment (Manarnz) [inline leaf-ui/js/object/evoObjects.js]: Why is `EVO.revertIntentionsText(graph.getElements(), paper);` repeated four times?

File: leaf-ui/js/object/evoObjects.js [hunk 4]
```diff
@@ -206,6 +167,10 @@ class EVO {
         EVO.refresh(analysisResult);
     }   
     
+     /**
+     * Switch between update modeling slider to analysis slider for EVO
+     *
```
Comment (Manarnz) [inline leaf-ui/js/object/evoObjects.js]: remove extra line

File: leaf-ui/js/object/evoObjects.js [hunk 5]
```diff
@@ -441,7 +406,7 @@ class EVO {
      * @param {*} intentionEval four digit code that corresponds to evidence pair (ex. 0011)
      */
     static getColor(intentionEval) {
-        if(IntentionColoring.isColorBlindMode) {
+        if(this.isColorBlindMode) {
```
Comment (Manarnz) [inline leaf-ui/js/object/evoObjects.js]: Add space between `if` and `(this.isColorBlindMode)`
Also, try to change it to `EVO.isColorBlindMode` for consistency

File: leaf-ui/js/object/evoObjects.js [hunk 6]
```diff
@@ -468,7 +433,16 @@ class EVO {
...
-        IntentionColoring.refresh(analysisResult);
+        EVO.refresh(analysisResult);
+    }
+
+    /**
+     * Toggles color blind mode
+     * @param {*} isTurningOnColorBlindMode 
+     */
+     static toggleColorBlindMode(isTurningOnColorBlindMode, analysisResult) {
+        this.isColorBlindMode = isTurningOnColorBlindMode;
```
Comment (Manarnz) [inline leaf-ui/js/object/evoObjects.js]: Try to change it to `EVO.isColorBlindMode` for consistency

File: leaf-ui/js/onFunctions.js [hunk 7]
```diff
@@ -738,34 +736,31 @@ function setInteraction(interactionValue) {
...
+    	var intention = curr.get('intention');
+        var initSatVal = intention.getUserEvaluationBBM(0).get('assignedEvidencePair'); 
+
+    	if (initSatVal === '(no value)') {
+            curr.attr('.satvalue/text', '');
+
+    	} else {
+            curr.attr('.satvalue/text', satisfactionValuesDict[initSatVal].satValue);
+    	}
+        //curr.attr({text: {fill: 'black'}});
```
Comment (Manarnz) [inline leaf-ui/js/onFunctions.js]: delete line

File: leaf-ui/rappid-extensions/ConfigInspector.js [hunk 8]
```diff
@@ -203,7 +203,7 @@ var Config = Backbone.View.extend({
     switchConfig: function () {
         $('#modelingSlider').css("display", "");
         $('#analysisSlider').css("display", "none");
-        IntentionColoring.refresh(undefined)
+        EVO.refresh(undefined)
```
Comment (Manarnz) [inline leaf-ui/rappid-extensions/ConfigInspector.js]: Add `;`

**Review by Manarnz — APPROVED** [review]
Looks good!

**Review by amgrubb — APPROVED** [review]
Approved to merge into ma-issue221

**Pull request merged**